src/main/java/org/mifos/identityaccountmapper/api/implementation/BatchAccountLookupApiController.java

```diff
@@ -19,7 +20,7 @@ public class BatchAccountLookupApiController implements BatchAccountLookupApi {
     @Autowired
     AccountLookupService accountLookupService;
     @Override
-    public ResponseEntity<ResponseDTO> batchAccountLookup(String callbackURL, RequestDTO requestDTO, String registeringInstitutionId) throws ExecutionException, InterruptedException {
+    public ResponseEntity<ResponseDTO> batchAccountLookup(String callbackURL, BatchAccountLookupRequestDTO requestDTO, String registeringInstitutionId) throws ExecutionException, InterruptedException {
```

Comment:
This API is async so we have to return RequestDTO, BatchAccountLookupRequestDTO will be sent in callback